pkgs/applications/networking/cluster/argocd/use-go-module.patch

```diff
@@ -0,0 +1,3058 @@
+diff --git a/Gopkg.lock b/Gopkg.lock
```

Comment:
Is generating this patch reproducible? Should we include an `update.sh` in this directory to get it to generate a patch on the fly?

Reply:
This is not reproducible at the moment. I had to manually fix broken dependencies in `go.mod` after doing `go mod init`

Reply:
Fair enough. Let's submit the patch to the upstream project then.